automatic/kubernetes-kompose/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,26 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageName = 'kubernetes-kompose'
+
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+$installLocation = "$(Get-ToolsLocation)\$packageName"
+
+$packageArgs1 = @{
+  PackageName    = $packageName
+  FileFullPath64 = Get-Item $toolsPath\*.exe.tar.gz
+  Destination    = $installLocation
+}
+Get-ChocolateyUnzip @packageArgs1
+
+$packageArgs2 = @{
+  PackageName    = $packageName
+  FileFullPath64 = Get-Item $installLocation\*.exe.tar
+  Destination    = $installLocation
+}
+Get-ChocolateyUnzip @packageArgs2
```

Comment:
wasn't this fixed in a later version of chocolatey?
So you didn't have to extract it twice, could be useful to make a check if it's already extracted or not.